web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -5,19 +5,17 @@
 odoo.define('web_translate_dialog.translate_dialog', function(require){
 "use strict";
 
-var _ = require('_');
-var $ = require('$');
+/*global Reflect*/
+/*eslint no-undef: "error"*/
 
 var core = require('web.core');
-var data = require('web.data');
 var common = require('web.form_common');
-
-var FormView = require('web.FormView');
-var View = require('web.View');
+var data = require('web.data');
 var Dialog = require('web.Dialog');
-
-var _t = core._t;
+var FormView = require('web.FormView');
 var QWeb = core.qweb;
+var _t = core._t;
+var View = require('web.View');
```

Comment:
Why this diff? You are doing the same, but it was alphabetically sorted before and now it is not... 🤔